homeworks/05-css-advanced/styles.css

```diff
@@ -0,0 +1,451 @@
+    --text-terciary: #19191B;
+}
+
+[data-theme="dark"] {
+    --bg-primary: #0C151D;
+    --bg-secondary: #171F26;
+    --bg-terciary: #0C151D;
+    --bg-details: #FFE071;
+    --text-primary: #F1F2F4;
+    --text-secondary: #A3ABB2;
+    --text-terciary: #A3ABB2;
+}
+
+body {
+    margin: 0;
+}
+
+h1,
+h2,
+h3,
+p,
+span,
+.section__link {
+    font-weight: 500;
+    font-family: "Poppins", sans-serif;
+}
+
+.wrapper {
+    background-color: var(--bg-primary);
+    transition: all 2s linear;
+}
+
+.theme-transition {
+    transition: all 2s linear;
+}
+
+.container {
+    min-height: 100vh;
+    max-width: 1300px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem 3rem;
+    gap: 2rem;
+}
+
+.header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 3rem;
+}
+
+.header__button {
+    width: 5rem;
+    height: 5rem;
+    border-radius: 50%;
+    border: none;
+    outline: none;
+    background-color: var(--bg-secondary);
+    cursor: pointer;
+}
+
+.header__button:hover {
+    animation-name: button-animation;
+    animation-duration: .8s;
+    animation-timing-function: ease;
+    animation-iteration-count: infinite;
+}
+
+.header__button-svg,
+.header__main-svg {
+    fill: var(--text-primary);
+}
+
+.header__main {
+    display: flex;
+    align-items: center;
+}
+
+.header__main-img {
+    width: 12.5rem;
+    height: 12.5rem;
+    border-radius: 50%;
+    object-fit: cover;
+    padding: 0.6rem;
+    background-color: #FFFFFF;
+    border: 4px solid var(--bg-details);
+}
+
+.header__main-developer {
+    padding: 1rem 0 1rem 2rem;
+}
+
+.header__title,
+.section__stats-title {
+    color: var(--text-primary);
+}
+
+.header__subtitle,
+.section__stats-p,
+.footer__p {
+    color: var(--text-secondary);
+}
+
+.header__title {
+    font-size: 2.875rem;
+    margin: 0;
+}
+
+.header__subtitle,
+.section__stats-p,
+.section__link,
+.main__portfolio-text,
+.main__skills,
+.footer__p,
+.main__image-text {
+    font-size: 1.375rem;
+}
+
+.header__subtitle {
+    margin: 0;
+}
+
+.header__main-list {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+}
+
+.header__main-link svg {
+    transition: all .2s ease;
+}
+
+.header__main-link svg:hover {
+    transform: scale(2);
+}
+
+.section {
+    display: flex;
+    flex-direction: column;
+    gap: 3rem;
+}
+
+.section__stats {
+    display: flex;
+    justify-content: space-around;
+}
+
+.section__stats-item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.section__stats-title,
+.section__stats-p,
+.main__portfolio-text,
+.main__skills,
+.main__image-title,
+.main__image-p {
+    margin: 0;
+}
+
+.section__stats-title {
+    font-size: 1.5rem;
+}
+
+.section__secondary {
+    display: flex;
+    justify-content: center;
+    gap: 2rem;
+}
+
+.section__button {
+    width: 18.125rem;
+    padding: 1.438rem 0;
```

Comment:
You use too many 'rem'. Sizes and distances have to be just according to design - they are in pixels there!